handlers.go

```diff
@@ -44,13 +44,13 @@ func (a *Application) home(w http.ResponseWriter, r *http.Request) {
 	data.Render(w, r)
 }
 
-func (a *Application) showTaskReport(w http.ResponseWriter, r *http.Request) {
+func (s *Server) showTaskReport(w http.ResponseWriter, r *http.Request) {
 	if r.URL.Path != "/task/report" {
```

Comment:
It feels like these checks shouldn't be necessary, doesn't it? How could we even get here unless the mux routed here because this was the request URL?